dot/parachain/dispute/backend.go

```diff
@@ -0,0 +1,207 @@
+package dispute
+
+import (
+	"sync"
+	"time"
+
+	"github.com/ChainSafe/gossamer/dot/parachain/dispute/types"
+	parachainTypes "github.com/ChainSafe/gossamer/dot/parachain/types"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/dgraph-io/badger/v4"
+	"github.com/tidwall/btree"
+)
+
+// Backend is the backend for the dispute coordinator module.
+type Backend interface {
```

Comment:
Would it be beneficial to have an interfaces file here? I feel like if we plan to use mocks for testing it can be nice to keep the interfaces together

Reply:
Yeah, maybe. Since it isn't using mocks at the moment, how about we revisit this if it changes?